include/seqan3/range/container/concatenated_sequences.hpp

```diff
@@ -318,17 +359,17 @@ class concatenated_sequences
      *
      * Strong exception guarantee (no data is modified in case an exception is thrown).
      */
-    template <std::ranges::forward_range rng_type = value_type>
-    concatenated_sequences(std::initializer_list<rng_type> ilist)
+    template <std::ranges::forward_range value_type_t = value_type>
     //!\cond
-        requires is_compatible_value<rng_type>
+        requires is_compatible_with_value_type<value_type_t>
     //!\endcond
+    concatenated_sequences(std::initializer_list<value_type_t> ilist)
```

Comment:
```suggestion
    concatenated_sequences(std::initializer_list<value_type_t> ilist)
```

Ah now I found it, you meant this one.